internal/gcsx/bucket_manager_test.go

```diff
@@ -0,0 +1,122 @@
+package gcsx
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/googlecloudplatform/gcsfuse/internal/storage"
+	"github.com/jacobsa/gcloud/gcs"
+	"github.com/jacobsa/gcloud/gcs/gcsfake"
+	. "github.com/jacobsa/ogletest"
+	"github.com/jacobsa/timeutil"
+)
+
+func TestFlags(t *testing.T) { RunTests(t) }
+
+const TestBucketName string = "gcsfuse-default-bucket"
+
+////////////////////////////////////////////////////////////////////////
+// Boilerplate
+////////////////////////////////////////////////////////////////////////
+
+type BucketManagerTest struct {
+	bucket            gcs.Bucket
+	storageHandle     *storage.Storageclient
+	fakeStorageServer storage.FakeStorage
```

Comment:
just call it fakeStorage instead of fakeStorageServer

Reply:
fixed